python/Ganga/new_tests/GPI/Bugs/Savannah10064.py

```diff
@@ -5,7 +5,7 @@
 
 class Savannah10064(GangaUnitTest):
 
-    def Savannah10064(self):
+    def test_Savannah10064(self):
```

Comment:
We don't need to rename all these functions. The test runner is configured to find tests beginning with `Savannah` and `JIRA` so I'd rather keep these as they are.